org.apache.bookkeeper.bookie.TestEntryLog is very flaky (expecially on GitHub Actions machines)

org.apache.bookkeeper.bookie.TestEntryLog test fails very often on GitHub actions, this is very annoying for contributors

```
ERROR] Tests run: 28, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 10.731 s <<< FAILURE! - in org.apache.bookkeeper.bookie.TestEntryLog
[ERROR] testGetEntryLogsSet(org.apache.bookkeeper.bookie.TestEntryLog)  Time elapsed: 0.033 s  <<< FAILURE!
java.lang.AssertionError: expected:<[0, 1]> but was:<[0]>
	at org.apache.bookkeeper.bookie.TestEntryLog.testGetEntryLogsSet(TestEntryLog.java:442)

[ERROR] testGetEntryLogsSet(org.apache.bookkeeper.bookie.TestEntryLog)  Time elapsed: 0.003 s  <<< FAILURE!
java.lang.AssertionError: expected:<[0, 1]> but was:<[0]>
	at org.apache.bookkeeper.bookie.TestEntryLog.testGetEntryLogsSet(TestEntryLog.java:442)

[ERROR] testGetEntryLogsSet(org.apache.bookkeeper.bookie.TestEntryLog)  Time elapsed: 0.003 s  <<< FAILURE!
java.lang.AssertionError: expected:<[0, 1]> but was:<[0]>
	at org.apache.bookkeeper.bookie.TestEntryLog.testGetEntryLogsSet(TestEntryLog.java:442)
```

--- a/bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/TestEntryLog.java
+++ b/bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/TestEntryLog.java
@@ -67,13 +67,16 @@ import org.apache.commons.io.FileUtils;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.FixMethodOrder;
 import org.junit.Test;
+import org.junit.runners.MethodSorters;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Tests for EntryLog.
  */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class TestEntryLog {
     private static final Logger LOG = LoggerFactory.getLogger(TestEntryLog.class);
 

